src/components/Icons/iconChangelogData.ts

```diff
@@ -16,6 +16,13 @@ export const changelogData: ChangelogData[] = [
     affects: ["Documentation", "Functionality"],
     notes: ["Added the actionLightbulb icon."],
   },
+  {
+    date: "Prerelease",
+    version: "Prerelease",
+    type: "Update",
+    affects: ["Documentation", "Functionality"],
+    notes: ["Addsed the editorMode (pencil) icon."],
```

Comment:
This came in with the merge from `development`, but `Addsed` should be `Added`.  Want to fix it here?